widget/table.go

```diff
@@ -67,6 +89,50 @@ func (t *Table) CreateRenderer() fyne.WidgetRenderer {
 	return r
 }
 
+// SetSelected will mark the specified cell (at row, col) to be marked as selected.
+func (t *Table) SetSelected(row, col int) {
+	t.SelectedRow = row
+	t.SelectedColumn = col
+
+	t.scrollToVisible(row, col)
+	if t.OnCellSelected != nil {
+		t.OnCellSelected(row, col)
+	}
+
+	if t.moveCallback != nil {
+		t.moveCallback()
```

Comment:
Surely we shouldn't be moving the scroller on unselection either... but I will have to go back and look at the original code to see what moveCallback does.

Reply:
it updates the indicators around the border.

Reply:
Gotcha